docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+#### Inflation
+
+Increase in the [available supply](https://docs.decred.org/advanced/inflation/) of Decred as new DCR is minted into existence through the block reward.
+
+#### Live ticket
+
+Tickets that are waiting to be called.
+
+#### Mainnet
+
+The proper Decred network - a term used to differentiate from testnet.
+
+#### Mempool
+
+Transactions waiting to be mined.
+
+#### Miners
+
+Miners help secure the network via Proof-of-Work (PoW) mining. Miners recieve block rewards and transaction fees for successfully validating blocks. 
+
+#### Mining Pool
+
+A group of miners who share (pool) their computational resources to mine DCR. When a member of a mining pool successfully mines a block, the rewards are split among all pool members in proportion to the hashpower they contribute to the pool. 
+
+#### Missed ticket
+
+Tickets that have been called but did not respond - these can be revoked, but do not grant a reward.
+
+#### Orphan
+
+A valid block which is not built upon and therefore not included in the definitive blockchain.
+
+#### Politeia
+
+A system for facilitating the submission and discussion of [proposals](https://proposals.decred.org/) in an environment with transparent censorship. 
+
+#### Private key
+
+A string of data that allows you to access and spend DCR stored in address associated with the private key. When sending DCR from one address to another, the wallet must use the private key of the sending address to sign the transaction.
+
+#### Private passphrase
+
+A passphrate that is used to encrypt wallet.db files. wallet.db files encrypted with a private passphrase cannot be used without that passphrase.
+
+#### Proof-of-Stake (PoS) voting
+
+The mechanism by which ticket holders vote to approve blocks confirmed by PoW miners (thus providing a check on PoW miners), earn staking rewards, and vote on consensus rule changes and Politeia proposals. 
+
+#### Protocol rules
+
+See [consensus rules](#consensus-rules). 
+
+#### Public key
+
+A long string of alphanumeric characters generated algorithmically from a private key. The user’s public key is used to prove that a transaction was signed using their private key.
+
+#### Quorum
+
+The minimum level of participation required in order for a decision-making process to produce a valid outcome. Changes to the consensus rules require at least 10% of votes to be for or against the change in order to be valid. Politeaia proposals require 20%.
+
+#### Regular transaction tree
+
+The normal [transactions](https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/dgjsyxd) in a block: sending DCR to an address, coinbase transactions such as PoW Miner reward and Treasury stipend.
+
+#### Reorg
+A reorganization of the blockchain in which a set of blocks are replaced by another (longer) set. The number of blocks replaced is the depth of the reorg. 
```

Comment:
The reorganize to the best chain may not necessarily follow the longest chain.  The best chain is the one with the most work.

Reply:
Would it be more intuitively understood if we say fastest chain? I say this because in decred a minority fork can have more hashpower yet be reorganized to the one with the majority of tickets.  Unless work encompasses hashpower and stake....

Reply:
No, the chain with the most cumulative work wins.  Neither the number of tickets (so long as there are at least the minimum number required to satisfy the consensus rules that is), nor the number of blocks determine the ultimate winning branch (chain).

It is true that the number of tickets influence miner behavior, and the result is that there is strong preference towards creating blocks with more tickets, but that is something different.

Reply:
Reworded below. I am wondering though was "work" means, precisely. My googling has only turned up threads with developers using the term without explaining it. Is it just the sum of all difficulties in each block for the whole chain? Do we need to define work here (or elsewhere)?

#### Reorg

A reorganization (or reorg) of the blockchain is when a set of blocks are replaced by another set which has more work. The number of blocks replaced is the depth of the reorg.

Reply:
The rewording looks good.  To answer your question, work refers to the number of hashes that were required to produce the chain.  Since, the required difficulty changes periodically, the same number of blocks can have a different cumulative amount of total work due to differing requirements.